src/java/org/apache/cassandra/schema/DistributedMetadataLogKeyspace.java

```diff
@@ -149,7 +149,8 @@ public static boolean tryCommit(Entry.Id entryId,
         }
         catch (Throwable t)
         {
-            logger.error("Caught an exception while trying to CAS", t);
+            //TODO (now): add back... it spams the logs making the real errors get truncated
```

Comment:
will add back, way too much spam in the test logs making it hard to find *real* issues...